admin/src/js/controllers/edit-user.js

```diff
@@ -332,7 +332,13 @@ angular
                   $uibModalInstance.close();
                 })
                 .catch(function(err) {
-                  $scope.setError(err, 'Error updating user');
+                  if (err.status === 400) {
```

Comment:
I think it's better to check if the properties you are about to use in the translation exist instead of checking for the status code. For example:
```
if (err && err.data && err.data.error && err.data.error.translationKey) {
  ...
} else {
  ...
}
```